tensorflow/core/kernels/dml_conv_ops.cc

```diff
@@ -659,8 +659,7 @@ class DmlFusedConv2DKernel : public DmlKernel {
   REGISTER_KERNEL_BUILDER(                                               \
       Name("_FusedConv2D").Device(DEVICE_DML).TypeConstraint<type>("T"), \
       DmlKernelWrapper<DmlFusedConv2DKernel<type>, ConvShapeHelper>);
-// FusedConv2D only supports float32
-TF_CALL_float(DML_REGISTER_KERNEL);
+TF_CALL_DML_FLOAT_TYPES(DML_REGISTER_KERNEL);
 #undef DML_REGISTER_KERNEL
```

Comment:
Does this work? The opdef indicates that this only supports float and double.

Reply:
(similar comment applies to _FusedMatMul)

Reply:
Ah you're right, I didn't check the opdef. Strange restriction.... I'll revert to float only.